ANS_JS_template.tex

```diff
@@ -93,92 +93,92 @@
 
 
 In contrast to multiple means of energy generation producing one product, multiple input multiple output systems (MIMO) use any excess energy not used by the grid at a given moment to generate other products to increase profitability \cite {Garcia2013}. Figure 1 has been constructed to demonstrate the differences between a system where electricity generation is the sole purpose (MISO) versus a system that produces electricity and another good, such as desalinated water or hydrogen. The sources of energy are gray while the energy sinks are white. Both diagrams differ from co-generation because they illustrate more than one source of energy contributing to the system. The primary benefit of a MIMO system is a diversified income source, thus the system is better economically shielded from fluctuations in electricity prices. Both types of systems attempt to provide reliable energy to the grid while incorporating a fluctuating source of energy.
 
-\begin{figure*}
-\includegraphics[width=\textwidth]{MISO_MIMO.png}
-\caption{\small \sl This figure compares the MISO and MIMO configurations demonstrating the differences between traditional hybrid energy systems that are focused on generating reliable electricity and non-traditional hybrid energy systems which have the added objective of generating an additional product.  While many of the elements are the same, the MIMO system includes  Industrial Process which is either thermally or electrically coupled.}
-\end{figure*}
+ While the current microgrid hybrid energy system tools, such as HOMER, should be evaluated when determining the characteristics for a NRHES model, there are major differences between small, home-owner projects, and large industrial scale projects. The large industrial scale projects require more thorough physics models, especially when they are including an industrial process. There has been significant research done on small scale MISO HES.  There has been less work done on a large HES which has a major goal of functioning as a backup source to a fluctuating renewable. The focus of this paper is exclusively on the large HES.
```

Comment:
Changed the single image comparing MISO and MIMO systems to two stacked images.  I also included a clarifying description that this research is specifically on large HES.